### DISCUSSION TITLE
Facebook Orientation Problem with RTMP

### DISCUSSION THREAD

**Discussion by onur-ardal** [ant-media/Ant-Media-Server#5682] [Q&A] [answered]
Hi everyone,

We trouble in rtmp orientation problem for Facebook or another social media.

Antmedia Server : Enterprise Edition 2.5.3 20230107_0851
We are developing an app with Flutter, but we are not using the Ant Media Flutter library. Instead, we are directly developing with Flutter WebRTC.

The Flutter mobile app creates a live stream and starts correctly. Since the app is designed for mobile phones, we only use portrait mode, and the camera view is also in portrait orientation.
![image](https://github.com/ant-media/Ant-Media-Server/assets/80475909/886813f5-d46e-4b71-bf81-6529b42e8ac9)

It works correctly when connected to other phones and the Ant Media Server viewer.
![image](https://github.com/ant-media/Ant-Media-Server/assets/80475909/2dddd48f-55a3-4030-95c5-1ebc397cce46)

But in Facebook app displays camera view rotated landscape.
![image](https://github.com/ant-media/Ant-Media-Server/assets/80475909/f2fb83ac-9600-452b-856c-1095624c0603)

How could I fix this?

**Comment by tremzo** [comment]
@onur-ardal Let me see if I experience the same using Flutter SDK.

**Comment by USAMAWIZARD** [comment]
which version of flutter WebRTC Library are you using please use the latest version , As far as I know this should be resolved with the latest release , https://pub.dev/packages/ant_media_flutter/versions

**Reply by onur-ardal** [reply]
We do not use the Antmedia Flutter SDK, instead we connect directly to Antmedia with Flutter WebRTC library.

**Comment by onur-ardal** [comment, marked as answer]
Hi @USAMAWIZARD ,

We tried Antmedia Android library for mobile. And we got same result. 
Did you catch any bug or something?

**Reply by USAMAWIZARD** [reply]
@onur-ardal Yes I know this issue it was fixed recently on flutter SDK side , let me check again if I am able to reproduce the issue now.

**Reply by onur-ardal** [reply]
Hi Usama,

Your comment is right, We tried with antmedia flutter SDK and everything correct working our first look.
Thanks for help.
